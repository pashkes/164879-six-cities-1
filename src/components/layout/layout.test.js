import React from "react";
import {shallow, configure} from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import Layout from "./layout";

configure({adapter: new Adapter()});

it(`Should render is correctly`, () => {
  const layout = shallow(<Layout />);

  expect(layout).toMatchSnapshot();
});
