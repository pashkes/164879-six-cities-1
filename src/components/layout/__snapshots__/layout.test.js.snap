// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render is correctly 1`] = `
<SideEffect(DocumentTitle)>
  <div
    className=""
  >
    <ConnectFunction />
  </div>
</SideEffect(DocumentTitle)>
`;
